### PR TITLE
Fixes #8448: Add a tag to rudder-data to identify supported version

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8448

### CODE REVIEW

File: release-data/rudder-3.3.cson [hunk 1]
```diff
@@ -4,6 +4,9 @@ release = "3.3"
 
 released = false
 
+# Is this version currently supported?
+supported = true
```
Comment (peckpeck) [inline release-data/rudder-3.3.cson]: 3.3 is not currently supported

**Comment by amousset** [comment]
Commit modified

File: release-data/rudder-3.3.cson (hunk 1 above)
Comment (amousset) [inline release-data/rudder-3.3.cson]: Indeed!

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**